### ISSUE TITLE
Empty Table of content (TOC)

### ISSUE THREAD

**Issue by Pelu2k12** [xceedsoftware/DocX#286]
Hello,
I'm trying to generate a table of content, but the resulting document doesn't display any items in the TOC. I'm using your example code ([DocX/Examples/Samples/TableOfContent/TableOfContentSample.cs](https://github.com/xceedsoftware/DocX/blob/master/Examples/Samples/TableOfContent/TableOfContentSample.cs)) for the TOC without any modifications.

![image](https://user-images.githubusercontent.com/2618669/49095551-b7c38500-f236-11e8-91e8-249aa9b9d616.png)

**Comment by XceedBoucherS** [comment]
Hi,

When you open the resulting document with MS Word, you should see a popup asking if you want to update the document fields. Saying yes will update the TOC.

**Comment by Wayne-Mather** [comment]
@Pelu2k12 You may need to look at the fix I did in ticket 300 for the TOC to be indented.

[Issue #300](https://github.com/xceedsoftware/DocX/issues/300)

**Comment by wis3guy** [comment]
@XceedBoucherS is this the only way to generate a TOC? In a web app scenario the popupp is undesirable and the message that word presents say's nothing aboout updating the TOC. It only scares the users into clicking NO.

**Comment by XceedBoucherS** [comment]
@wis3guy yes, for now this is the way to create a TOC. What this "DocX" does is update the ooxml from the docx document. When MS Word opens this modified docx, he will ask to update the TOC. Saying yes will force MS Word to update its fields (which the TOC is) and display the TOC adequately.
